docs/guides/intro-to-graphql.md

```diff
@@ -308,7 +308,7 @@ query {
 
 ### `search`
 
-Will search the list to limit results.
+Will search the list to limit results. It can search only by [labelField](/docs/api/create-list.md#labelfield).
```

Comment:
This isn't quite true. At the moment the `search` is applied to a `Text` field called `name`. This is not currently configurable, although that is a feature which is on our backlog.